packages/components/inputs/rich-text-input/src/rich-text-input.visualspec.js

```diff
@@ -1,5 +1,20 @@
 import percySnapshot from '@percy/puppeteer';
+import puppeteer from 'puppeteer';
 import { getDocument, queries } from 'pptr-testing-library';
+let browser;
+let page;
+
+jest.setTimeout(20000);
```

Comment:
nit: you could also specify a custom timeout for specific tests: https://jestjs.io/docs/api#testname-fn-timeout

Reply:
Thanks. Yes, I started with that, but then I switched to defining it in one place for brevity. I thought this fine tuning of those values might be done as the last step if we don't come up with any better solution for fixing the visual tests.